ruma-events-macros/src/event_content.rs

```diff
@@ -302,11 +325,10 @@ fn generate_event_content_impl(ident: &Ident, event_type: &LitStr) -> TokenStrea
     }
 }
 
-fn needs_redacted(input: &DeriveInput) -> bool {
-    input
-        .attrs
-        .iter()
-        .flat_map(|a| a.parse_args::<EventMeta>().ok())
-        .find(|a| a == &EventMeta::CustomRedacted)
-        .is_none()
+fn needs_redacted(input: &[MetaAttrs]) -> bool {
+    input.iter().any(|a| !a.is_custom())
```

Comment:
Seems incorrect. Shouldn't this be `!any(is_custom)` rather than `any(!is_custom)`?

Reply:
Ah now I can actually explain it, earlier I was just trying stuff till it worked, so if an event content struct has the `#[ruma_event(custom_redacted)]` attr it `is_custom -> true` and it does __not__ `need_redacted` since it is manually implementeded e.g. aliases content.

Reply:
sorry I miss read the review. Yes, that is more clear and works fine.